sample-skill/lambda/lambda_function.py

```diff
@@ -78,7 +79,7 @@ def get_access_token() -> tuple:
 
 def get_course_progress(username: str, coursename: str) -> str:
     """Return the progress of an user in a course"""
-    access_token, error_message = get_access_token()
+    access_token, error_message = get_jwt_token()
```

Comment:
are you getting the JWT access token? Or a bearer token? where do we specify the JWT type?

Reply:
Currently, the Bearer token is obtained, I will rename the function.